src/mpc-hc/MainFrm.cpp

```diff
@@ -11189,6 +11189,10 @@ void CMainFrame::OpenSetupAudio()
 {
     m_pBA->put_Volume(m_wndToolBar.Volume);
 
+    if (AfxGetAppSettings().nCLSwitches & CLSW_VOLUME) {

```

Comment:
Without it, it doesn't set the Volume at all. Only on 2nd launch of mpc-hc. I don't know why and what I did wrong